css/css-align/gaps/META.yml

```diff
@@ -7,6 +7,6 @@ links:
         - test: gap-animation-003.html
         - test: gap-animation-004.html
     - url: http://example.org
+      label: example-label1
```

Comment:
If you need at least one label for testing purposes, can I suggest adding a "interop-2021" label to https://github.com/web-platform-tests/wpt.fyi/pull/2686? (And if not, dropping this label seems better.)

Reply:
I will drop this change

Reply:
```suggestion
```